pkg/services/star/star.go

```diff
@@ -0,0 +1,12 @@
+package star
+
+import (
+	"context"
+)
+
+type Service interface {
+	StarDashboard(ctx context.Context, cmd *StarDashboardCommand) error
+	UnstarDashboard(ctx context.Context, cmd *UnstarDashboardCommand) error
+	IsStarredByUserCtx(ctx context.Context, query *IsStarredByUserQuery) (bool, error)
+	GetUserStars(ctx context.Context, cmd *GetUserStarsQuery) (map[int64]bool, error)
```

Comment:
And I guess this map type should be `type GetUserStarsResult map[int64]bool` or better a struct with a map inside (like we agreed: Command/Cmd, Query, Result). I think for the bool result above we could make an exception.